src/docs/samples/code/application-plugin/basic/groovy/build.gradle

```diff
@@ -12,7 +12,7 @@ docker {
         baseImage = 'dockerfile/java:openjdk-7-jre'
         maintainer = 'Benjamin Muschko "benjamin.muschko@gmail.com"'
         ports = [9090, 5701]
-        tag = 'jettyapp:1.115'
+        tags = ['jettyapp:1.115']
```

Comment:
Maybe provide multiple tags here so that users will know how things work? Possibly the same comment for other doc-related changes.

Reply:
done